Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java

```diff
@@ -219,6 +220,17 @@ protected void testGetActiveParticipantDestination(LogEventRequestType request,
             webContextProperties.getProperty(NhincConstants.REMOTE_HOST_ADDRESS));
     }
 
+    protected void testAuditSourceIdentification(List<AuditSourceIdentificationType> auditSourceIdentification, AssertionType assertion) {
+        if (assertion != null && assertion.getUserInfo() != null && assertion.getUserInfo().getOrg() != null
+            && assertion.getUserInfo().getOrg().getHomeCommunityId() != null
+            && auditSourceIdentification != null && !auditSourceIdentification.isEmpty()
+            && auditSourceIdentification.size() > 0) {
```

Comment:
Two things:

1. isEmpty() and .size() > 0 is redundant (and always prefer .isEmpty())
2. isEmpty() isn't needed here; the enhanced for loop will gracefully take care of the "empty list" case.